src/components/notification/notification.ts

```diff
@@ -202,6 +202,7 @@ export class SbbNotificationElement extends LitElement {
   protected override render(): TemplateResult {
     const hasTitle = !!this.titleContent || this._namedSlots['title'];
 
+    setAttribute(this, 'role', 'status');
```

Comment:
Not sure if it is better to delegate this to the consumer so it doesn't have any default behavior. on the other hand, it seems a good default :). @kyubisation meaning?

Reply:
can it be overridden, e.g. with role alert from outside? I feel atm. every re render would override it again.

Reply:
My gut feeling would be to set this in the `connectedCallback`, if it is not yet set.
e.g. `this.role = this.role ?? 'status';` (we should be able to use `this.role` now, I think).

Reply:
I think there was something preventing using this.role. maybe tests or chromatic oder something else? we can try again ;) I could not find mdn docs so far. 
Anyways what my main concern is, is basically you have an enclosing container and apply role status there and watch there for changes (like appearing notification). But somehow it works only if notification itself has role status as well.
But in general I think it's ok to predefine role status as long as there is an option to override it.

Reply:
The same could be achieved with `this.setAttribute('role', this.getAttribute('role') ?? 'status')`, if we can't use this.role yet, I guess.

Reply:
I'd prefer the `this.setAttribute('role', this.getAttribute('role') ?? 'status')` because it's at one place

Reply:
You can completely move the role attribute setting into connectedcallback